xarray/convert.py

```diff
@@ -291,7 +291,7 @@ def from_iris(cube):
     cube_data = cube.core_data() if hasattr(cube, "core_data") else cube.data
 
     # Deal with dask and numpy masked arrays
-    if isinstance(cube_data, dask_array_type):
+    if is_duck_dask_array(cube_data):
```

Comment:
is it possible to have `iris` cubes with `duck dask arrays`?

Reply:
I do not know, but it's a good point. If `duck dask arrays` are not specifically supported by `iris`, maybe this should be changed back to the `isinstance` check?